tensorflow/core/kernels/mkl_relu_op.cc

```diff
@@ -1023,7 +1023,8 @@ class MklReluGradOpBase : public OpKernel {
       }
 
       OP_REQUIRES_OK(context, context->forward_input_or_allocate_output(
-                                  {diff_dst_index}, diff_src_index,
+                                  {static_cast<const int>(diff_dst_index)},
```

Comment:
Wouldn't static_cast<int> suffice?

Reply:
Not sure, I just did what all other function calls were doing in the same file.

Reply:
Oh I see those variables are defined as const size_t. Nevermind.